shard/src/ext/pug_cog.py

```diff
@@ -0,0 +1,54 @@
+from discord.ext import commands
+from contextlib import suppress
+import time
+import asyncio
+import discord
+from lib.config import logger
+
+
+class Pug(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def startpug(self, ctx, role: discord.Role, requiredPlayers: int):
```

Comment:
would this be better as a string than a role?  it looks like we don't really use anything from the role, and it might be restrictive

Reply:
Maybe we would want to have config options in the future where certain roles would have a default amount of required players... Maybe I'm reaching too much for this

Reply:
decided to remove this since its never used